custom_components/spotcast/helpers.py

```diff
@@ -349,7 +349,7 @@ def search_tracks(
 def add_tracks_to_queue(
     spotify_client: spotipy.Spotify, tracks: list = [], limit: int = 20
 ):
-    filtered = list(filter(lambda x: x["type"] == "track", tracks))
+    filtered = list(filter(lambda x: x and x["type"] == "track", tracks))
```

Comment:
I would prefer we test specifically for `dict` type and we should also use the `get` method to ensure cases where the key is potentially missing. Should look like:

```python
filtered = list(filter(lambda x: isinstance(x, dict) and x.get("type") == "track", tracks))
```

Reply:
This is an edge case for None value, both implementations should be fine for me.